graylog2-server/src/test/java/org/graylog2/NettyTest.java

```diff
@@ -0,0 +1,31 @@
+package org.graylog2;
```

Comment:
Please run `./mvnw license:format` to add the missing license headers. Thanks!

Reply:
Done.